avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -27,69 +27,69 @@ Update the given readme file's 'Resource Types' section based on the given templ
 #>
 function Set-ResourceTypesSection {
 
-    [CmdletBinding(SupportsShouldProcess)]
-    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', 'ResourceTypesToExclude', Justification = 'Variable used inside Where-Object block.')]
-    param (
-        [Parameter(Mandatory)]
-        [hashtable] $TemplateFileContent,
-
-        [Parameter(Mandatory)]
-        [object[]] $ReadMeFileContent,
-
-        [Parameter(Mandatory = $false)]
-        [string] $SectionStartIdentifier = '## Resource Types',
-
-        [Parameter(Mandatory = $false)]
-        [string[]] $ResourceTypesToExclude = @('Microsoft.Resources/deployments')
-    )
-
-    # Process content
-    $SectionContent = [System.Collections.ArrayList]@(
-        '| Resource Type | API Version |',
-        '| :-- | :-- |'
-    )
-
-    $RelevantResourceTypeObjects = Get-NestedResourceList $TemplateFileContent | Where-Object {
-        $_.type -notin $ResourceTypesToExclude -and $_
-    } | Select-Object 'Type', 'ApiVersion' -Unique | Sort-Object Type -Culture 'en-US'
-
-    $ProgressPreference = 'SilentlyContinue'
-    $VerbosePreference = 'SilentlyContinue'
-    foreach ($resourceTypeObject in $RelevantResourceTypeObjects) {
-        $ProviderNamespace, $ResourceType = $resourceTypeObject.Type -split '/', 2
-        # Validate if Reference URL is working
-        $TemplatesBaseUrl = 'https://learn.microsoft.com/en-us/azure/templates'
-        try {
-            $ResourceReferenceUrl = '{0}/{1}/{2}/{3}' -f $TemplatesBaseUrl, $ProviderNamespace, $resourceTypeObject.ApiVersion, $ResourceType
-            $null = Invoke-WebRequest -Uri $ResourceReferenceUrl
+  [CmdletBinding(SupportsShouldProcess)]
```

Comment:
Does this change impact existing readme files for any published modules?

Reply:
@eriqua - VSCode had auto format enabled!